databricks/koalas/tests/test_dataframe.py

```diff
@@ -5422,3 +5422,31 @@ def test_align(self):
             pdf_l, pdf_r = pdf1.align(pdf2, join=join, axis=1)
             self.assert_eq(kdf_l.sort_index(), pdf_l.sort_index())
             self.assert_eq(kdf_r.sort_index(), pdf_r.sort_index())
+
+    def test_between_time(self):
+        i = pd.date_range("2018-04-09", periods=4, freq="1D20min")
+        ts = pd.DataFrame({"A": [1, 2, 3, 4]}, index=i)
+        kts = ks.DataFrame({"A": [1, 2, 3, 4]}, index=i)
```

Comment:
We usually use `pdf` for pandas DataFrame and `kdf` for Koalas DataFrame. Could you follow it?

Reply:
btw, you can use `kdf = ks.from_pandas(pdf)`.